src/features/GetCommands.ts

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+// tslint:disable:no-console
+import * as vscode from "vscode";
+import { LanguageClient, RequestType } from "vscode-languageclient";
+import { IFeature } from "../feature";
+
+export const GetCommandsRequestType = new RequestType<any, any, void, void>("powerShell/getCommands");
```

Comment:
You should be able to slim down this file by specifying Command instead of any. It should automatically cast it.

Like this:
https://github.com/PowerShell/vscode-powershell/blob/master/src/features/Console.ts#L10

But I'm not 100%. It would save a lot of code I think.

Reply:
I'm not sure how to do this if possible. I attempted to replace the any with Command and it crashes PSES among other things.